### PR TITLE
Feature/header smaller clickable

### PR DESCRIPTION
**Pull request by SammySteiner**

### CODE REVIEW

**Comment by SammySteiner** [comment]
[Header: reduce logo size and make clickable](https://trello.com/c/oiUxaQ1t/50-header-reduce-logo-size-and-make-clickable)

File: src/components/layout/layout.module.css [hunk 1]
```diff
@@ -27,6 +32,7 @@
 .headerImg {
   width: 3.5rem;
   height: 2.5rem;
+
```
Comment (vonbearshark) [inline src/components/layout/layout.module.css:35]: nit +

**Review by vonbearshark — COMMENTED** [review]
Everything lgtm, but what's up with the lockfile changes?

**Review by vonbearshark — APPROVED** [review]

File: src/components/layout/layout.js [hunk 2]
```diff
@@ -21,16 +21,18 @@ export default function Layout({ children }) {
...
-          className={styles.headerImg}
-          src="../../images/logo.png"
-          alt="logo"
-        />
-        <Link to="/">
-          <h1>{site.siteMetadata.title}</h1>
-          <h2>Performance Comparison</h2>
+        <Link to="/" className={styles.logoWithText}>
+          <StaticImage
+            src="../../images/logo.png"
```
Comment (vonbearshark) [inline src/components/layout/layout.js:26]: Can we rm the old one? Or replace it with this new one? (right now it's in `images/`). No preference

**Review by vonbearshark — APPROVED** [review]

**Pull request merged**